README.md

```diff
@@ -27,6 +27,7 @@ The following items are supported:
 - Customizable storage backends
 - PID file writing
 - Logging
+- Configurable storage backend
```

Comment:
I think this is an awesome feature and really useful.

We should highlight it by making 3 sections here:
1. Features supported (and compatible) with nodejs configurable-http-proxy
2. Additional features not available in nodejs configurable-http-proxy
3. Features from nodejs configurable-http-proxy not supported (yet)

Reply:
Updated the README accordingly